common/src/main/resources/locales/de-de.yml

```diff
@@ -9,7 +9,7 @@ teleporting_action_bar_cancelled: '[Teleportation abgebrochen!](#ff7e5e)'
 teleporting_other_complete: '[%1% wurde zu %2% teleportiert.](#00fb9a)'
 teleporting_other_complete_position: '[%1% wurde zu ⚐ x: %2%, y: %3%, z: %4% teleportiert.](#00fb9a)'
 teleporting_all_players: '[Teleportiere](#00fb9a) [%1%](#00fb9a bold) [Spieler zu deiner Position…](#00fb9a)'
-teleporting_random_generation: '[Finding a safe random position to teleport you to…](#00fb9a)'
+teleporting_random_generation: '[Eine zufällige sichere Position um dich zu teleportieren, wird gesucht...](#00fb9a)'
```

Comment:
Can you replace the three-period ellipse you added (`...`) with the single-character UTF-8 horizontal ellipse, as is the case with other strings (`…`)?